lib/semian.rb

```diff
@@ -296,7 +296,19 @@ def require_keys!(required = [], **options)
   end
 end
 
+def hostname
+  v = %w(KUBE_HOSTNAME KUBE_HOST_NAME KUBE_NODENAME KUBE_NODE_NAME NODENAME NODE_NAME HOSTNAME HOST_NAME)
+  var = v.find { |x| ENV.include?(x) }
+  ENV[var] if var
+end
+
+def force_host_circuits
```

Comment:
This should be `force_host_circuits?` to be more Ruby-like.

Reply:
Done.